apps/web/playwright/api/management/contacts.spec.ts

```
@@ -0,0 +1,198 @@
+import { expect } from "@playwright/test";
+import { test } from "../../lib/fixtures";
+import { loginAndGetApiKey } from "../../lib/utils";
+
+test.describe("API Tests for Single Contact Creation", () => {
+  test("Create and Test Contact Creation via API", async ({ page, users, request }) => {
+    let environmentId, apiKey;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)